storage/framework/onboarding/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "stacks-onboarding",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@clack/core": "^0.3.4",
+    "@clack/prompts": "^0.7.0",
+    "picocolors": "^1.0.0"
+  }
+}
```

Comment:
This shouldn't be here and extracted to core packages, placed where appropriate, like the cli package.

We may already may require these deps elsewhere too